serverless/stacks/AppStack.ts

```diff
@@ -11,6 +11,7 @@ export function AppStack({ stack }: StackContext) {
       ),
     },
   });
+  /*
```

Comment:
why is this comment here? @alantao912

Reply:
I had to remove some code that was causing issues.

Reply:
got it! lets fix this to have idempotency as discussed in dm